Portfolio/index.css

```diff
@@ -1 +1,53 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+.main{
+    padding: 10px;
+    box-sizing: border-box;

```

Comment:
Consider breaking down the background-image property onto multiple lines for better readability.